ee-security/pom.xml

```diff
@@ -250,14 +218,11 @@
                     <plugin>
                         <groupId>org.wildfly.plugins</groupId>
                         <artifactId>wildfly-maven-plugin</artifactId>
-                        <version>${version.wildfly.maven.plugin}</version>
                         <configuration>
                             <feature-packs>
-                                <feature-pack>
-                                    <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
+                                <feature-pack>                                  <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
```

Comment:
Just a minor comment, but looks like the formatting is off here and below.

Reply:
Fixed!